src/main/java/gift/controller/GiftController.java

```diff
@@ -18,8 +18,12 @@ public GiftController(GiftRepository giftRepository) {
 
     @PostMapping("")
     public ResponseEntity<?> addGift(@RequestBody RequestGift requestGift) {
+<<<<<<< step#2
+        return new ResponseEntity<>(giftRepository.save(requestGift), HttpStatus.CREATED);
+=======
         giftRepository.save(requestGift);
         return new ResponseEntity<>(HttpStatus.CREATED);
+>>>>>>> sunja-an
```

Comment:
해결되지 않은 git 충돌이 남아있네요! 원하는 부분만 선택해서 해결할 수 있어요~
```suggestion
        return new ResponseEntity<>(giftRepository.save(requestGift), HttpStatus.CREATED);
```